collections/ansible_collections/demo/patching/roles/build_report_network/vars/main.yml

```diff
@@ -1 +1,12 @@
-file_path: /var/www/html
\ No newline at end of file
+#file_path: "{{ web_path | default('/var/www/html/reports') }}"
+file_path: "{{ web_path | default('/var/www/html') }}"
```

Comment:
file path default to `/var/www/html/reports` will match what is configured by `report_server` role

Reply:
Ok. I'm going back to `/var/www/html/reports` this time around